#163519758 Implement login feature API

mcaleb808 opened the pull request:
#### What does this PR do?
This PR will add login endpoint to system

#### Description of Task to be completed?
checking if a user exists by email
compare provided email to hashed email

#### How should this be manually tested?
1. npm install
2. connect Database
3. $ sequelize db:migrate
4. check on http://localhost:3000/api/v1/auth/login with POST method

#### What are the relevant pivotal tracker stories?
#163519758

Commit:
feat(authorization): Try to debug travis

- Add  to tests

[Starts #163519757]

mcaleb808 committed:
feature(Authorization) user login endpoint

checking if user exist by email
compare provided email to hashed email

[Delivers #163519758]

oesukam commented:
@mcaleb808 Can you correct the title of the pull request following the engineering play book and try to pull or rebase with the develop branch then push again.

Don't forget that you don't need to create another pull request for this branch, you can only update your changes

Commit:
feat(authorization): Fix conflic

- Fix conflict

[Starts #163519758]

Commit:
feat(authorization): Remove test.only from tests

- Remove only attributes to tests

[Starts #163519758]

mcaleb808 committed:
feature(Authorization) user login endpoint

checking if user exist by email
compare provided email to hashed email

[Delivers #163519758]

NkFab commented:
I think it's better to display the endpoint under the Description of the task to be completed for instance
The following endpoint should be working
`POST /auth/login`

Commit:
feat(authorization): Correct ababel version

- Add @babel/cli package

[Delivers #163519758]
